src/node_file.cc

```diff
@@ -969,14 +970,78 @@ static void InternalModuleReadJSON(const FunctionCallbackInfo<Value>& args) {
   Local<Value> return_value[] = {
     String::NewFromUtf8(isolate,
                         &chars[start],
-                        v8::NewStringType::kNormal,
+                        NewStringType::kNormal,
                         size).ToLocalChecked(),
     Boolean::New(isolate, p < pe ? true : false)
   };
   args.GetReturnValue().Set(
     Array::New(isolate, return_value, arraysize(return_value)));
 }
 
+Isolate* _isolate = nullptr;
+std::vector<Local<Value>>* _exports = nullptr;
+std::vector<Local<Value>>* _reexports = nullptr;
```

Comment:
These as static global state is not ideal and definitely is not threadsafe. By adding the `void* userdata` argument that I suggested, you can avoid this by creating a struct that holds these per operation.